pkg/rng/entropy.go

```diff
@@ -0,0 +1,139 @@
+package rng
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	trngList map[string]int
+)
+
+const (
+	hwRandomDir                string        = "/sys/class/misc/hw_random"
```

Comment:
we may want these to become public variables instead of private constants, to facilitate unit testing